lib/git.sh

```
@@ -153,19 +182,50 @@ show_entry() {
+        echo "$json" | jq .
+      else
+        printf '%s\n' "$json"
+      fi
+    fi
+    if git notes --ref="$NOTES_REF" show "$target" >/dev/null 2>&1; then
+      git notes --ref="$NOTES_REF" show "$target"
+    fi
+    return
+  fi
+  # ... rest of function
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)